Source/Core/InputCommon/InputConfig.cpp

```diff
@@ -117,3 +117,17 @@ bool InputConfig::ControllersNeedToBeCreated() const
 {
 	return m_controllers.empty();
 }
+
+bool InputConfig::IsControllerAKeyboard(int index) const
```

Comment:
That's really vague. I wouldn't understand what "regular controller" meant if I came across it.